datastream-server/src/main/java/com/linkedin/datastream/server/assignment/StickyMulticastStrategy.java

```diff
@@ -122,12 +124,19 @@ public StickyMulticastStrategy(Optional<Integer> maxTasks, Optional<Integer> imb
             .map(c -> c.stream().filter(x -> x.getTaskPrefix().equals(dg.getTaskPrefix())).collect(Collectors.toList()))
             .orElse(Collections.emptyList());
 
+        allAliveTasks.addAll(foundDatastreamTasks);
+
         if (!foundDatastreamTasks.isEmpty()) {
           newAssignment.get(instance).addAll(foundDatastreamTasks);
           currentAssignmentCopy.get(instance).removeAll(foundDatastreamTasks);
           numTasks -= foundDatastreamTasks.size();
         }
       }
+      // As SticyAssignment, we want to recycle task from dead instances
```

Comment:
nit: typo Sticky

Reply:
Thanks. I have run through the spell check with a too in the latest version